doc/nrf/protocols/bt_mesh/concepts.rst

```diff
@@ -316,3 +316,12 @@ Replay protection
 
 To guard against malicious devices replaying previous messages, every device keeps a running sequence number, which is used for outbound messages.
 Each mesh message is sent with a unique pair of sequence number and source address. When receiving a message, the receiving device stores the sequence number and makes sure that it is more recent than the last sequence number it received from the same source address.
+
+.. _mesh_concepts_coexistence:
+
+Bluetooth LE and mesh coexistence
+*********************************
+
+Bluetooth mesh uses the preset local identity value. For more information about advertisement identity, see :ref:`zephyr:bluetooth_mesh_core`.
```

Comment:
May be not easy to understand what this means. Maybe add a link to `BT_ID_DEFAULT`?

Reply:
There is reference to `BT_ID_DEFAULT` in upstream doc. I just added here reference to upstream. I tried to avoid documentation duplication in upstream and here. Do you think it is worth to add reference to `BT_ID_DEFAULT` either despite it presented in upstream?

Reply:
Added link here: https://github.com/zephyrproject-rtos/zephyr/pull/59356
Please review.